### PR TITLE
Update to use DataVolumes for imports

### PR DESCRIPTION
**Pull request by awels**
- Updated import as VM(linux/windows) to use DV.
- Updated improt as Template(linux/windows) to use DV.

Signed-off-by: Alexander Wels <awels@redhat.com>

### CODE REVIEW

**Comment by awels** [comment]
@aglitke Please review, this should work, but I haven't verified the template functionality because of https://github.com/kubevirt/kubevirt/issues/1507

**Comment by awels** [comment]
@aglitke I have verified this works for import as VM and Template flows.

**Comment by awels** [comment]
@aglitke Review please.

File: roles/import-from-url/tasks/deprovision.yml [hunk 1]
```diff
@@ -27,6 +27,3 @@
     - command: "oc patch clusterservicebroker template-service-broker -p '{\"spec\": {\"relistRequests\": {{ current_relist_count.stdout|int + 1 }} }}'"
       ignore_errors: yes
   when: image_type == "template"
-  
-- name: Delete PVC
```
Comment (aglitke) [inline roles/import-from-url/tasks/deprovision.yml:31]: Don't you need to delete the DV when deprovisioning?

Comment (awels) [reply roles/import-from-url/tasks/deprovision.yml:31]: No because we are deleting the VM, it cascades the deletion of the DV, and thus the PVC as well.

Comment (aglitke) [reply roles/import-from-url/tasks/deprovision.yml:31]: What about in the template case?

Comment (awels) [reply roles/import-from-url/tasks/deprovision.yml:31]: You make a good point about the template case, that will leave the DV behind. Let me fix that.

**Comment by aglitke** [comment]
> * Updated import as VM(linux/windows) to use DV.
> * Updated improt as Template(linux/windows) to use DV.
> 
> Signed-off-by: Alexander Wels [awels@redhat.com](mailto:awels@redhat.com)

This bumps the requirements of CDI and kubevirt.  Please update the Readme to mention which releases of CDI and kubevirt are required for this to work.  (I realize that this blocks this PR until we have those releases and that is intended) :)

**Pull request merged**